### PR TITLE
Clarified steps to setup the discord bot

### PR DESCRIPTION
**Pull request by FastestMolasses**
Updated the steps to properly setup the discord bot from scratch.

### CODE REVIEW

File: discord-bot/README.md [hunk 1]
```diff
@@ -62,19 +62,34 @@ cd oasst-shared
...
 # edit .env and add your bot token and other values
 # BOT_TOKEN is given by the discord developer portal when you create a bot
-# DECLARE_GLOBAL_COMMANDS is the id of the server where you added the bot (right click on the server icon and copy id)
+# DECLARE_GLOBAL_COMMANDS is the id of the server where you added the bot (right click on the server icon and copy id. Developer mode must be enabled.)
 # OWNER_ID can be leave as an empty list
 
 python -V  # 3.10
 
+# Create a virtual environment
+python -m venv venv
```
Comment (bitplane) [inline discord-bot/README.md:81]: This should be `.venv` like in the backend devcontainer

Comment (bitplane) [reply discord-bot/README.md:81]: Actually won't this create it in the wrong directory?

File: discord-bot/.gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 *.egg-info/
 __pycache__/
 
+env/
 .venv
```
Comment (bitplane) [inline discord-bot/.gitignore:6]: Not needed.

File: discord-bot/README.md (hunk 1 above)
Comment (bitplane) [inline discord-bot/README.md:81]: Should be .venv

File: discord-bot/README.md [hunk 3]
```diff
@@ -62,19 +62,34 @@ cd oasst-shared
...
+
+# Activate the virtual environment
+# (Windows)
+env\Scripts\activate
```
Comment (bitplane) [inline discord-bot/README.md:85]: Inconsistent with above instructions

**Review by bitplane — CHANGES_REQUESTED** [review]

**Comment by andreaskoepf** [comment]
Closing this since we will remove the classic python data collection bot from the project.

**Pull request closed without merging**